modules/gitops-argo-cd-notification.adoc

```diff
@@ -47,8 +47,7 @@ oc patch argocd openshift-gitops -n openshift-gitops --type merge --patch '{"spe
 ====
 
 [role="_additional-resources"]
-[id="additional-resources_argo-cd-notifications"]
-== Additional resources
+.Additional resources
 
-* To learn more about how to configure notifications, see link:https://argoproj.github.io/argo-cd/operator-manual/notifications/[Argo CD notifications controller].
+* To learn more about how to configure notifications, see link:https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/[Argo CD notifications controller].
```

Comment:
There is a link for release-2.10 published yesterday. Not sure if you want to use that link ? https://argo-cd.readthedocs.io/en/release-2.10/operator-manual/notifications/